examples/cmake-examples/QmlBarcodeGenerator/cmake/Locations.cmake

```diff
@@ -0,0 +1,61 @@
+set(COMPANY "Scythe Studio")
+set(COPYRIGHT "Copyright (c) 2022 Scythe Studio. All rights reserved.")
+set(IDENTIFIER "com.scythestudio.scodes.example")
+
+# ---CONFIGURATION---
+# MOST IMPORTANT PART -> change to ON/OFF if you want to use these elements in your project
+option(USE_QML "Add QML support" ON)
+#option(USE_TESTS "Include tests" ON )
+#option(USE_DOCUMENTATION "Add documentation based on Doxygen template" ON )
+#option(USE_TRANSLATIONS "Enable using translations" ON )
+option(USE_LIBS "Use external libraries" ON)
+
+#option(WINDOWS_TARGET "Set Windows as the target platform" ON )
+#option(LINUX_TARGET "Set Linux as the target platform" OFF )
+
+set(QT_NAME Qt5)
+#message("Support for Windows is ${WINDOWS_TARGET}")
+#message("Support for Linux is ${LINUX_TARGET}")
+
+# Target specific files
+#set(PLATFORM_DIR "platforms")
+#set(WINDOWS_PLATFORM_DIR "windows")
+#set(LINUX_PLATFORM_DIR "linux")
+
+# Locations - directories in project structure
+set(LIB_DIR "../../../src")
```

Comment:
Such type of path should be avoided - you should rather start with variables like `${CMAKE_SOURCE_DIR}`